src/lib/icon/icon.md

```diff
@@ -90,7 +90,8 @@ In thinking about accessibility, it is useful to place icon use into one of thre
 1. **Decorative**: the icon conveys no real semantic meaning and is purely cosmetic.
 2. **Interactive**: a user will click or otherwise interact with the icon to perform some action.
 3. **Indicator**: the icon is not interactive, but it conveys some information, such as a status.
-
+4. **Inline**: the icon is not interactive and is inlined into a block of text to convey 
```

Comment:
I would prefer not to add a fourth category, since it makes it seem more complicated overall. I think we could do:
```markdown
3. **Indicator**: the icon is not interactive, but it conveys some information, such as a status. This
includes using the icon in place of text inside of a larger message.
```